core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -103,37 +111,76 @@
     private ColumnType() {
     }
 
+    public static int geohashWithPrecision(int bits) {
+        return GeoHashExtra.setBitsPrecision(ColumnType.GEOHASH, bits);
+    }
+
+    public static int tagOf(int type) {
+        return type & 0xFF;
+    }
+
     public static int columnTypeOf(CharSequence name) {
         return nameTypeMap.get(name);
     }
 
+    // TODO: overload to take a CharSink and avoid having to create one at each call
```

Comment:
Let's statically add all 64 geohash types as pre-defined strings instead of using char sink

Reply:
i may have added too many